core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -1,144 +1,693 @@
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6">
+          <div class="row container donate-page-section-heading">
+            <span>
+              {{ 'I18N_DONATE_PAGE_HEADING_1' | translate }}
+              <span class="oppia-donate-page-text-color">
+                {{ 'I18N_DONATE_PAGE_HEADING_2' | translate }}
+              </span>
+            </span>
+          </div>
+          <div class="row">
+            <div class="col-sm-6">
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">
+                  1,500,000+
+                </div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_1' | translate }}</div>
+              </div>
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">
+                  2,000
+                </div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_2' | translate }}</div>
+              </div>
+            </div>
+            <div class="col-sm-6">
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">
+                  1
+                </div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_3' | translate }}</div>
+              </div>
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">
+                  100+
+                </div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_4' | translate }}</div>
+              </div>
+            </div>
+          </div>
+          <div class="row mt-5">
+            <div class="col-sm-6">
+              <a class="btn oppia-donate-page-watch-video-btn"
+                 href="https://youtu.be/OConyxG7HaM">
+                {{ 'I18N_DONATE_PAGE_WATCH_VIDEO_BUTTON_TEXT' | translate }}
+              </a>
+            </div>
+            <div class="col-sm-6">
+              <!-- TODO(#16643): Update Blog URL after blog migration is complete -->
+              <a class="btn oppia-donate-page-read-blog-btn"
+                 href="https://medium.com/oppia-org">
+                {{ 'I18N_DONATE_PAGE_READ_BLOG_BUTTON_TEXT' | translate }}
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <picture class="oppia-donate-page-learner-img-container">
+            <source type="image/webp"
+                    [srcset]="getImageSet('/donate/oppia-donate-learner', 'webp')">
+            <source type="image/png"
+                    [srcset]="getImageSet('/donate/oppia-donate-learner', 'png')">
+            <img [src]="getStaticImageUrl('/donate/oppia-donate-learner', 'png')"
```

Comment:
Angular CLI started complaining that `getStaticImageUrl` only takes in one argument but two are supplied here 😛.

CLI is unhappy about the following: 
```
Error: core/templates/pages/donate-page/donate-page.component.html:109:75 - error TS2554: Expected 1 arguments, but got 2.

109             <img [src]="getStaticImageUrl('/donate/oppia-donate-learner', 'png')"
                                                                              ~~~~~

  core/templates/pages/donate-page/donate-page.component.ts:41:16
    41   templateUrl: './donate-page.component.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component DonatePageComponent.


Error: core/templates/pages/donate-page/donate-page.component.html:205:67 - error TS2554: Expected 1 arguments, but got 2.

205                   <img [src]="getStaticImageUrl('/splash/Gaurav', 'png')"
                                                                      ~~~~~

  core/templates/pages/donate-page/donate-page.component.ts:41:16
    41   templateUrl: './donate-page.component.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component DonatePageComponent.


Error: core/templates/pages/donate-page/donate-page.component.html:230:65 - error TS2554: Expected 1 arguments, but got 2.

230                   <img [src]="getStaticImageUrl('/splash/mira', 'png')"
                                                                    ~~~~~

  core/templates/pages/donate-page/donate-page.component.ts:41:16
    41   templateUrl: './donate-page.component.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component DonatePageComponent.


Error: core/templates/pages/donate-page/donate-page.component.html:255:68 - error TS2554: Expected 1 arguments, but got 2.

255                   <img [src]="getStaticImageUrl('/splash/Dheeraj', 'png')"
```

Reply:
From the context of the html file, I have changed it to  `getImageSet`

Reply:
Thanks for catching this. I will probably have a follow-up PR that fixes an issue with the new page that we just found. I can include this fix in there, but yearh changing to getImageSet is the way to go.

Reply:
Just an update, I've fixed the issues in https://github.com/oppia/oppia/pull/16650.